mne/time_frequency/tfr.py

```diff
@@ -4208,9 +4308,39 @@ def _prep_data_for_plot(
     freqs = freqs[freq_mask]
     # crop data
     data = data[..., freq_mask, :][..., time_mask]
-    # complex amplitude → real power; real-valued data is already power (or ITC)
+    # handle unaggregated multitaper (complex or phase multitaper data)
+    if taper_weights is not None:  # assumes a taper dimension
+        logger.info("Aggregating multitaper estimates before plotting...")
+        if np.iscomplexobj(data):  # complex coefficients → power
+            data = _tfr_from_mt(data, taper_weights)
+        else:  # tapered phase data → weighted phase data
+            data = (data * taper_weights[np.newaxis, :, :, np.newaxis]).mean(axis=1)
```

Comment:
This is my guess at aggregating over tapers for phase data. Can anyone confirm if this is correct?

Reply:
@larsoner this is more your wheelhouse... 🙈

Reply:
Just to help the next person
```suggestion
        else:  # tapered phase data → weighted phase data
            # channels, tapers, freqs, time
            assert data.ndim == 4
            # weights as a function of (tapers, freqs)
            assert taper_weights.ndim == 2
            data = (data * taper_weights[np.newaxis, :, :, np.newaxis]).mean(axis=1)
```